### ISSUE TITLE
image classification

### ISSUE THREAD

**Issue by julien-ur** [julien-ur/ITT-Abschlussprojekt#9]

**Comment by julien-ur** [comment]
neural network or svm

**Comment by tran-tuananh** [comment]
Created neural network (nn) to train and classify *.npy files from https://console.cloud.google.com/storage/browser/quickdraw_dataset/full/numpy_bitmap/?pli=1
Currently using CPU-only version, as GPU version currently not working correctly.
Training not completed.
Experimentation on nn settings needed, to provide fast training and good results (different layers/combinations/nodes/filter kernel sizes/batch_sizes)

**Comment by tran-tuananh** [comment]
Classifier (convoluted neural network) was trained with 15 categories, with 20000 images for each, for 10 epochs (due to time and performance constraints). 2000 test images for each category.
Resulting accuracy: ~92%
